pyproject.toml

```diff
@@ -1,11 +1,12 @@
 [tool.poetry]
-name = "action-merge-checks"
+name = "merge-checks"
 version = "0.1.0"
 description = "GitHub Merge Checks"
 authors = ["moneymeets GmbH <service@moneymeets.com>"]
 
 [tool.poetry.dependencies]
 python = "^3.10"
+PyGithub = "^1.55"
```

Comment:
```suggestion
PyGithub = "*"
```

Reply:
Changed in edc201d88ed0f486262a8bb45f546c37c7d58525.